pkg/services/accesscontrol/migrator/zanzana.go

```diff
@@ -111,9 +114,9 @@ func managedPermissionsCollector(store db.DB) TupleCollector {
 		for _, p := range permissions {
 			var subject string
 			if len(p.UserUID) > 0 {
-				subject = zanzana.NewObject(zanzana.TypeUser, p.UserUID)
+				subject = zanzana.NewObject(zanzana.TypeUser, p.UserUID, "")
 			} else if len(p.TeamUID) > 0 {
-				subject = zanzana.NewObject(zanzana.TypeTeam, p.TeamUID)
+				subject = zanzana.NewObject(zanzana.TypeTeam, p.TeamUID, "member")
```

Comment:
I am not sure this is correct. This should just set up tuple from a managed permission to either a user and team. So member should not be there right. E.g. in the case of folder -> team

Reply:
hm thinking a bit about it this makes more sense, all members of that team can reach this (or admin).